flow/FastAlloc.h

```diff
@@ -266,4 +266,24 @@ inline void freeFast(int size, void* ptr) {
 	delete[](uint8_t*) ptr;
 }
 
+[[nodiscard]] inline void* allocateFast4kAligned(int size) {
+	if (size <= 4096)
```

Comment:
I'm assuming this was the fix for the memory overhead from aligned_alloc?
If so it would likely be useful to add a comment explaining why this was done.
Is the assumption that with 20k+ aligned alloc, the extra 4k for the malloc metadata is a small enough overhead?

Reply:
Ok, and yes.